documentation/asciidoc/computers/getting-started/network-installation.adoc

```diff
@@ -51,15 +51,15 @@ You will need a keyboard to make use of the network installation feature. While
 
 video::b1SYVpM9lto[youtube]
 
-You should also insert a **blank** SD Card into the Raspberry Pi's SD Card slot.
+You should also insert a **blank** SD Card into Raspberry Pi computers SD Card slot.
 
 NOTE: You should make sure there isn't a bootable SD Card, or USB drive, attached to your Raspberry Pi.
 
-Power on your Raspberry Pi. As always it will first look for an SD Card, and then a USB drive, to find bootable media. However if you have a keyboard attached the Raspberry Pi will now show the network installation screen.
+Power on your Raspberry Pi. As always, it will first look for an SD Card, and then a USB drive, to find bootable media. However, if you have a keyboard attached, Raspberry Pi will now show the network installation screen.
```

Comment:
First and third commas shouldn't have been added. Removing the "the" means that the second sentence needs reworded, it can't just be removed.